crypto/hash.go

```diff
@@ -64,3 +97,6 @@ func NewSHA3_256() Hasher {
 func NewSHA3_384() Hasher {
 	return hash.NewSHA3_384()
 }
+
+// NewKeccak_256 returns a new instance of Keccak256 hasher.
+var NewKeccak_256 = hash.NewKeccak_256
```

Comment:
I was wondering if I can update all the functions of type :
```
func SdkFunction() {
    flow_go_crypto.Funtion()
}
```
to 
```
var  SdkFunction = flow_go_crypto.Funtion
```
Is there a reason the first type was preferred so far?

Reply:
I don't know the initial design decisions (so hopefully Pete will speak to that) but this proposed change I guess has a downside of the possibility to overwrite that value saved there. Also, the call stack would be different but that I don't think has much of purpose anyway.

Reply:
Overwriting is a good point, though I think there is a low risk for a developer to overwrite a that value. 
For the call stack, since this is a short function, the complier might be already inlining it. 

I'll just keep the original code :)

Reply:
Honestly, I'm fine either way, I just wanted to provide my answer to the question.